code/modules/reagents/reagents/food.dm

```diff
@@ -406,28 +406,36 @@
 
 /datum/reagent/consumable/larvajelly
 	name = "Larva Jelly"
-	description = "The blood and guts of a xenomorph larva blended into a paste. Drinking this is bad for you."
+	description = "A hormonal, sticky, slick jelly made inside xeno's reproductory organs, containing nutrients and pheromones to make the host's body nurish and nurture the larva inside."
 	reagent_state = LIQUID
-	nutriment_factor = 0
-	color = "#66801e"
-	taste_description = "burning"
+	nutriment_factor = 1
```

Comment:
Same as above, `nutriment_factor` is also `1` on `/datum/reagent/consumable`, so it can be removed from here.